homeassistant/components/minecraft_server/config_flow.py

```diff
@@ -85,3 +89,38 @@ def _show_config_form(self, user_input=None, errors=None) -> FlowResult:
             ),
             errors=errors,
         )
+
+    async def async_is_server_online(self, config_data: dict[str, Any]) -> bool:
```

Comment:
```suggestion
    async def _async_is_server_online(self, host: str, port:int) -> bool:
```

Mark the function as protected and we have already the host and the port separated at the caller, so we pass it directly.